### ISSUE TITLE
Nav bar collapse button

### ISSUE THREAD

**Issue by evmallidis** [strapi/strapi#22526]
### Required System information

- Node.js version: 20.12.1
- NPM version: 10.8.1
- Strapi version: 5.5.0
- Database: mysql
- Operating system: ubuntu
- Is your project Javascript or Typescript: js

### Describe the bug

After upgrade to v5 the collapsible side nav button is missing. It's always minified. 
Is this intentionally changed?

Thank you

**Comment by markkaylor** [comment]
I believe relying on icons + tooltips instead of a collapsible sidenav was intentional, I'll let @lucasboilly confirm and close the issue if so.

**Comment by derrickmehaffy** [comment]
Yeah we got rid of the collapsible menu and just keep it collapsed always.

I'll go ahead and close this one.

**Comment by arilanto** [comment]
unbelievable !

**Comment by Marshall-Hallenbeck** [comment]
> Yeah we got rid of the collapsible menu and just keep it collapsed always.

Why did you remove this exactly? I liked having the text in the sidebar so I don't have to remember what the small icons mean.

**Comment by simotae14** [comment]
if you hover over the icon, you get a tooltip with the icon label

**Comment by lucasboilly** [comment]
Hi @Marshall-Hallenbeck :)

We did it with content editors in mind. Since in most projects they only have access to a few items (like Home, Content Manager, Media Library and Settings) we thought it was worth it to keep the collapsed version only to:
- Give more space to the fields in the edit view and make content edition more pleasant.
- Make Strapi a little bit more responsive so it can be used better in small devices (pretty common among content editors) or in split screen. We're still far from being 100% responsive but that's a first step.

Since most of the time spent in Strapi during a project is spent by content editors we also need to think of them. The idea is to continue to improve the product while making the UI simpler for non-technical people.

Hope it answers your question and thanks for sharing your feedback! We'll keep this in mind to see how we could improve the navigation in the future. Maybe a shorter animation on the tooltips would be a quick-win?

**Comment by Marshall-Hallenbeck** [comment]
I understand simplicity, but for people who don't use the UI that much, it's much more confusing to remember which buttons are which. Before there was a toggle to close the sidebar; couldn't you just have that but default it to be closed, and allow the user to toggle it on and off (and remember what the last setting was)?

**Comment by SalahAdDin** [comment]
> Hi [@Marshall-Hallenbeck](https://github.com/Marshall-Hallenbeck) :)
> 
> We did it with content editors in mind. Since in most projects they only have access to a few items (like Home, Content Manager, Media Library and Settings) we thought it was worth it to keep the collapsed version only to:
> 
>     * Give more space to the fields in the edit view and make content edition more pleasant.
> 
>     * Make Strapi a little bit more responsive so it can be used better in small devices (pretty common among content editors) or in split screen. We're still far from being 100% responsive but that's a first step.
> 
> 
> Since most of the time spent in Strapi during a project is spent by content editors we also need to think of them. The idea is to continue to improve the product while making the UI simpler for non-technical people.
> 
> Hope it answers your question and thanks for sharing your feedback! We'll keep this in mind to see how we could improve the navigation in the future. Maybe a shorter animation on the tooltips would be a quick-win?

Are you guys sure about that much simplicity is intuitive for content managers?

**Comment by lucasboilly** [comment]
For now we're not planning to bring this back as it saves a lot of space in the UI. This initiative was discussed and tested by content editors before being shipped.

**Comment by Marshall-Hallenbeck** [comment]
> For now we're not planning to bring this back as it saves a lot of space in the UI. This initiative was discussed and tested by content editors before being shipped.

Why wouldn't you just default closed and allow people to open it? Makes no sense. I'm upgrading an app to use the newer version and I'm definitely going to get complaints.

It's 2025, what sorta people don't have large monitors?

**Comment by SalahAdDin** [comment]
> > For now we're not planning to bring this back as it saves a lot of space in the UI. This initiative was discussed and tested by content editors before being shipped.
> 
> Why wouldn't you just default closed and allow people to open it? Makes no sense. I'm upgrading an app to use the newer version and I'm definitely going to get complaints.
> 
> It's 2025, what sorta people don't have large monitors?

There are people who has no big screens. But it should be more like a feature, not just collapsed all the time.

**Comment by sisirkon** [comment]
> Hi [@Marshall-Hallenbeck](https://github.com/Marshall-Hallenbeck) :)
> 
> We did it with content editors in mind. Since in most projects they only have access to a few items (like Home, Content Manager, Media Library and Settings) we thought it was worth it to keep the collapsed version only to:
> 
> * Give more space to the fields in the edit view and make content edition more pleasant.
> * Make Strapi a little bit more responsive so it can be used better in small devices (pretty common among content editors) or in split screen. We're still far from being 100% responsive but that's a first step.
> 
> Since most of the time spent in Strapi during a project is spent by content editors we also need to think of them. The idea is to continue to improve the product while making the UI simpler for non-technical people.
> 
> Hope it answers your question and thanks for sharing your feedback! We'll keep this in mind to see how we could improve the navigation in the future. Maybe a shorter animation on the tooltips would be a quick-win?

I have run into the issue with smaller screen / split screen screens. so collapsable has helped. but, it would be nice, to have the option of it being collapsable by default and expanded as needed.

The number of icons by default are 6 and can increase based on the plugins introduced (this would be a use case specific).

Also, text is easier to understand than icons. It takes away any guess work involved with navigation. Since, same icon can be interpreted differently based on applications used across organizations. 

With responsive, in mind , as the screen sizes increases (laptop to monitor even a regular 24 inch or just laptop screens with higher resolution) the icon size do not change based on making it not ideal for selection.

**Comment by arilanto** [comment]
Hello,
To manage this (huge) regression, I add extra js and css in app.tsx bootstrap in order to put back collapse button (almost work, but need to refresh page after login to make it totally work)
Cheers

**Comment by SalahAdDin** [comment]
> Hello, To manage this (huge) regression, I add extra js and css in app.tsx bootstrap in order to put back collapse button (almost work, but need to refresh page after login to make it totally work) Cheers

any PR? so we can check it? You could check how was it handled in Strapi 4 to solve that small issue.

**Comment by Khaleel** [comment]
> Hello, To manage this (huge) regression, I add extra js and css in app.tsx bootstrap in order to put back collapse button (almost work, but need to refresh page after login to make it totally work) Cheers

Please can you share the code you did. I find it pretty naive and arrogant that the Strapi team decided to do this. Left navigation bars are in Shopify, Stripe and almost all major Sass platforms online on the planet. Content Editors are all now savvy and usually work in marketing; don't insult intelligence. Y

**Comment by bap72190** [comment]
This really needs to be revisited. Why not default to collapsed but allow developers to add back in expanded option.. this is not intuitive and is very confusing for users and adds extra frustration where something so simple should be available.

**Comment by SalahAdDin** [comment]
> This really needs to be revisited. Why not default to collapsed but allow developers to add back in expanded option.. this is not intuitive and is very confusing for users and adds extra frustration where something so simple should be available.

It seems there is a plugin to make it back as before...

**Comment by bap72190** [comment]
@SalahAdDin do you happen to know where this plugin can be found?

**Comment by SalahAdDin** [comment]
> [@SalahAdDin](https://github.com/SalahAdDin) do you happen to know where this plugin can be found?

Unfortunately, no, it should be in this issue or in another one.

**Comment by SalahAdDin** [comment]
> Hello, To manage this (huge) regression, I add extra js and css in app.tsx bootstrap in order to put back collapse button (almost work, but need to refresh page after login to make it totally work) Cheers

Any snipped for this?
